test/integration/http2_integration_test.h

```diff
@@ -4,7 +4,7 @@
 #include "test/test_common/test_base.h"
```

Comment:
Remove the includes as well?

It might make more sense to git rollback for most files, and then do a follow-up clean up PR, but your call.

Reply:
FInished changing includes; my inclination was that master had moved on, so I felt it was easier just to sed them back, which worked fine and didn't really have exceptions.